api-proxy/src/reqhandler.mjs

```diff
@@ -0,0 +1,136 @@
+import * as fs from 'fs'
+const fsp = fs.promises
+import * as https from 'https'
+import * as stream from 'stream'
+import * as nu from 'util'
+import * as zlib from 'zlib'
+
+const omit = ks => obj =>
+  Object.keys(obj).reduce((acc, k) => ks.includes(k) ? acc : ({...acc, [k]: obj[k]}), {})
+
+const promiseEvent = eventName => x => new Promise(resolve => {
```

Comment:
would "convert event to promise" be an accurate way to describe what this is doing?

Reply:
I don't think so. We're not transforming the event in any way. Instead, we're just waiting for it using the promise mechanism instead of a callback. "Promise event" here means, "promise me that you'll get an event in the future." I'm using promise in its verb form to return one in its noun form. Does that context make you think this name is a good choice?

Reply:
I'm not following.  Not too surprising, since I don't really understand how events work.  Also: I find curried functions to be an order of magnitude more difficult to read, so there's that too.

But this might be moot: I don't see that this is being used anywhere.  Maybe it can be removed?